src/elements/content-uploader/__tests__/Footer.test.js

```diff
@@ -0,0 +1,59 @@
+import React from 'react';
+import { shallow } from 'enzyme';
+import noop from 'lodash/noop';
```

Comment:
move lodash before enzyme

Reply:
I think we try to follow a convention where react packages are at the top, and we alphasort the rest